src/esp/assets/managers/AttributesManagerBase.h

```diff
@@ -89,7 +89,14 @@ class AttributesManager {
    */
   virtual AttribsPtr createDefaultAttributesTemplate(
       const std::string& templateName,
-      bool registerTemplate = false) = 0;
+      bool registerTemplate = false) {
+    // create default attributes descriptor
+    AttribsPtr attributes = initNewAttribsInternal(templateName);
+    if (nullptr == attributes) {
+      return attributes;
+    }
+    return this->postCreateRegister(attributes, registerTemplate);
+  }
```

Comment:
Creating a default template is effectively the same for every template - just make and initialize a new one, verify it is not null, then call to conditional register and return template.